addons/website/static/src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,14 @@
+header {
+    .o_main_navbar {
+        .o_website_systray {
+            --NavBar-entry-margin-right: #{$o-navbar-entry-padding-h};
+        }
```

Comment:
I don't know why the web app is that generic about this but I doubt we should add any margin, only paddings. Especially not as a website exception. Worse: there are actually spaces *between* button which actually click on the button somehow:
- Hover the publish button -> you have an hover effect
- Go towards the mobile button -> you have a space between the two buttons where you don't have an hover effect
- Click -> the publish button is enabled :fearful: 

Note: I don't know if this is a bug that this PR creates, my intuition is that the PR *reveals* it adding margins with the generic `--NavBar-entry-margin-right` stuff. Maybe this requires a deeper framework-level fix.

Reply:
This is on me 🙋‍♂️ (sorry @rdeodoo ).
I've switched to enterprise and applied some changes, then I've forgot to test community again .

I'll fix this and https://github.com/odoo/odoo/pull/131228#discussion_r1296884756 as well

Reply:
Ok perfect, I was actually changing the selector and replying to the other QSM review but I'll let you do it :+1: